src/app/hooks/useGlean.ts

```diff
@@ -62,6 +67,11 @@ export const useGlean = (experimentationId?: string) => {
     // Instead of the specific type definitions we generated in the npm script
     // `build-glean-types`, Glean takes a non-specific "ExtraArgs" type as
     // parameter to `record`.
+
+    // Record the `plan_tier` key on all events.
+    // @ts-expect-error Not present in map, but exists on all events.
```

Comment:
@Vinnl I see the `plan_tier` key in the generated types, but I'm not sure how to use convince typescript that this is OK since it's not in the `GleanMetricMap`. Do you have any thoughts?